go.mod

```diff
@@ -45,13 +45,13 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	// Please ensure that you update the image tags in `examples/operating-system-manager.yaml` as well.
 	k8c.io/operating-system-manager v1.0.0
-	k8s.io/api v0.24.2
+	k8s.io/api v0.25.0
```

Comment:
is it really necessary to upgrade k8s deps?

Reply:
Why shouldn't we?

Reply:
Because MC doesn't exist in isolation and other projects (who import MC) would be forced to upgrade too, which is not always optimal or wanted.

Reply:
Well, I don't see any harm in upgrading them as well. Since we need to keep them updated to the latest packages anyways :)

Reply:
>we need to keep them updated to the latest packages anyways 

false

Reply:
So for KKP and KubeOne, we don't want to use the latest deps for k8s API in the master? What's the reason for not wanting to keep them updated?

Reply:
Anyways, if someone wants to use old deps then they can always pin them. :)